aws/asg.go

```diff
@@ -322,31 +304,47 @@ func hasTagsASG(tags []*autoscaling.TagDescription, expectedTags map[string]stri
 	return true
 }
 
+func attachTargetGroupsToAutoScalingGroup(svc autoscalingiface.AutoScalingAPI, targetGroupARNs []string, autoScalingGroupName string) error {
+	// limit target group updates to 10 at a time since this is the limit
+	// in AWS API.
+	return processChunked(targetGroupARNs, 10, func(chunk []string) error {
```

Comment:
nit :nail_care: since this is the "limit" from AWS I think a constant would fit us well.

Reply:
`10` is a constant ;)

Reply:
I have added a link to the docs that specify the limit